src/fetch.js

```diff
@@ -1,7 +1,7 @@
 const BASE_URL = "https://api.github.com";
 
 export function searchUsers(userName) {
-  return fetch(BASE_URL + "/mmit search/users?q=" + userName)
+  return fetch(BASE_URL + "/search/users?q=" + userName)
```

Comment:
of course i will accept it but at the same time such cases should never be repeated in future. check your code attentively before committing